action/action-dsl/src/test/java/com/powsybl/action/dsl/ActionDslLoaderTest.java

```diff
@@ -151,6 +215,54 @@ private void addPhaseShifter() {
                     .setAlpha(5.0)
                     .setRho(6.0)
                 .endStep()
+                .beginStep()
+                    .setR(1.0)
+                    .setX(2.0)
+                    .setG(3.0)
+                    .setB(4.0)
+                    .setAlpha(5.0)
+                    .setRho(6.0)
+                .endStep()
+                .beginStep()
+                    .setR(1.0)
+                    .setX(2.0)
+                    .setG(3.0)
+                    .setB(4.0)
+                    .setAlpha(5.0)
+                    .setRho(6.0)
+                .endStep()
                 .add();
     }
+
+    private static final class DeltaTapData {
+
+        private final int iniTapPosition;
+        private final int expectedTapPosition;
+        private final int deltaTap;
+        private final String testName;
+
+        private DeltaTapData(int iniTapPosition, int expectedTapPosition, int deltaTap, String testName) {
+            this.iniTapPosition = iniTapPosition;
+            this.expectedTapPosition = expectedTapPosition;
+            this.deltaTap = deltaTap;
+            this.testName = testName;
+        }
+
+        private int getIniTapPosition() {
```

Comment:
Even for test classes, I think we have to make method or field names human readable. I'm not sure **ini** is the standard word or abbreviation for initial